source/includes/_endpoint-dataset.md

```diff
@@ -795,8 +806,8 @@ See ["Expressions"](#expressions) for more on Crunch expressions.
 The following rules apply for all formats:
 
 * The dataset's exclusion filter will be applied; however, any of the user's personal "applied filters" are not, unless they are explicitly included in the request.
-* Hidden/discarded variables are not exported.
-* Personal (private) variables are not exported.
+* Hidden/discarded variables are not exported unless editors use a `where` clause, then it will be evaluated over all non hidden variables.
```

Comment:
This is not clear and sounds self-contradicting. I think you mean that you won't get hidden variables unless you request them explicitly (or with *), and even then only if you have edit privileges on the dataset.